content/en/apps/examples/supervisor-reference-app.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Supervisor Reference Application"
+linkTitle: "Supervisor Reference Application"
+weight:
+description: >
+  Supervisor Reference Application
+relatedContent: >
+  apps/concepts
+  apps/reference/forms/app
+  apps/reference/tasks
+---
+
+{{% pageinfo %}}
+
+A reference app for CHW supervisors to support performance management of CHWs using a mobile app.
+
+[Medic](https://medic.org/) has worked with [D-tree International](https://www.d-tree.org/) to build a CHT supervisor reference app. The app supports community health worker (CHW) supervisors to continually monitor and improve the program quality for Zanzibar National community health program Jamii ni Afya. The supervisor reference application is designed to enable supervisors to access CHW performance information and any other information required to supervise, mentor, and support CHWs to provide quality community health services. This reference app provides an example that CHT app developers can easily customize to meet the needs for their specific program areas to support CHW program management.
+
+You can find the code of this application in the CHT code repository on Github {link}
+
+{{% /pageinfo %}}
+
+## Problem being addressed
+Community Health Workers (CHWs) play a critical role in delivering quality care as part of the integrated primary health system, in some settings CHWs often serve as the only connection between the health system and the vulnerable and remote populations. For many community health programs, CHWs face a lot of challenges, this includes significant workloads, delivering health services to large and dispersed communities while being supported by limited and inadequate supervision. To ensure sustained positive impacts on CHW programmes, there is a need for supervisory interventions to be embedded within the broader community health system strengthening. The  supervisor reference app empowers supervisors with the ability to monitor in real time the performance of CHWs which can guide supervisors to support CHWs in provision of community health care services.
+
+## Solution overview
+The supervisor reference app workflows enables supervisors to: provide quality assurance supervisory activities, plan and document CHW supervisory activities, support CHW activities and monitor CHW performance in real time. The supervisor reference app has been designed to:
+<ul>
+<li>Monitor the performance of the CHW and supervisors using the in-app aggregate target and the supervisor targets functionalities respectively.</li>
+
+<li>Document and track CHW and supervisory activities.</li>
+<li>Schedule CHW monthly meetings quality monitoring visits and follow up visits.</li>
+<li>Support supervisors to report on health outcomes for their supervisory area.</li>
+<li>Identify CHWs who have not been visited by a supervisor.</li>
+<li>Follow up on CHW who have been inactive for 3 months.</li>
+</ul>
```

Comment:
please use markdown instead of HTML:

```suggestion
 * Monitor the performance of the CHW and supervisors using the in-app aggregate target and the supervisor targets functionalities respectively.
 * Document and track CHW and supervisory activities.
 * Schedule CHW monthly meetings quality monitoring visits and follow up visits.
 * Support supervisors to report on health outcomes for their supervisory area.
 * Identify CHWs who have not been visited by a supervisor.
 * Follow up on CHW who have been inactive for 3 months.
```